native/SampleApps/AuthFlowTester/AuthFlowTester/Views/UserCredentialsView.swift

```diff
@@ -28,103 +28,188 @@
+    public static let beaconChildConsumerSecret = "Beacon Child Consumer Secret"
+    
+    // Other fields
+    public static let additionalOAuthFields = "Additional OAuth Fields"
+}
+
 struct UserCredentialsView: View {
     @Binding var isExpanded: Bool
     let refreshTrigger: UUID
     
-    // Section expansion states - all start collapsed
-    @State private var userIdentityExpanded = false
-    @State private var oauthConfigExpanded = false
-    @State private var tokensExpanded = false
-    @State private var urlsExpanded = false
-    @State private var communityExpanded = false
-    @State private var domainsAndSidsExpanded = false
-    @State private var cookiesAndSecurityExpanded = false
-    @State private var beaconExpanded = false
-    @State private var otherExpanded = false
+    // Section expansion states - all expand/collapse together with parent
+    @State private var userIdentityExpanded = true
+    @State private var oauthConfigExpanded = true
+    @State private var tokensExpanded = true
+    @State private var urlsExpanded = true
+    @State private var communityExpanded = true
+    @State private var domainsAndSidsExpanded = true
+    @State private var cookiesAndSecurityExpanded = true
+    @State private var beaconExpanded = true
+    @State private var otherExpanded = true
+    
+    // Export alert state
+    @State private var showExportAlert = false
+    @State private var exportedJSON = ""
     
     var body: some View {
         VStack(alignment: .leading, spacing: 8) {
-            Button(action: {
-                withAnimation {
-                    isExpanded.toggle()
+            HStack {
+                Button(action: {
+                    withAnimation {
+                        isExpanded.toggle()
+                        // Expand/collapse all subsections with the parent
+                        expandAllSubsections(isExpanded)
+                    }
+                }) {
+                    HStack {
+                        Text("User Credentials")
+                            .font(.headline)
+                            .fontWeight(.semibold)
+                            .foregroundColor(.primary)
+                        Spacer()
+                        Image(systemName: isExpanded ? "chevron.up" : "chevron.down")
+                            .font(.caption)
+                            .foregroundColor(.secondary)
+                    }
                 }
-            }) {
-                HStack {
-                    Text("User Credentials")
-                        .font(.headline)
-                        .fontWeight(.semibold)
-                        .foregroundColor(.primary)
-                    Spacer()
-                    Image(systemName: isExpanded ? "chevron.up" : "chevron.down")
-                        .font(.caption)
-                        .foregroundColor(.secondary)
+                Button(action: {
```

Comment:
New button to export all the JWT details in a JSON.
It's used in UI tests (faster than collecting all the static texts).